.eslintrc

```diff
@@ -0,0 +1,152 @@
+    "react/jsx-closing-bracket-location": 1,
+    "react/jsx-curly-spacing": [2, "always"],
+    "react/jsx-indent-props": [1, 2],
+    "react/jsx-no-undef": 1,
+    "react/jsx-uses-react": 1,
+    "react/jsx-uses-vars": 1,
+    "react/jsx-wrap-multilines": 1,
+    "react/react-in-jsx-scope": 1,
+    "react/prefer-es6-class": 1,
+    // no binding functions in render for perf
+    "react/jsx-no-bind": 1,
+
+    // handle async/await functions correctly
+    // handle object spread
+    "babel/object-shorthand": 1,
+
+    // handle potential errors
+    "brace-style": [2, "1tbs", { "allowSingleLine": true }],
+    "comma-dangle": [2, "always-multiline"],
+    "consistent-return": 0,
+    "indent": [2, 2, {"SwitchCase": 1}],
+    "keyword-spacing": [2],
+    "max-len": [1, 100, 2, {"ignoreComments": true}],
+    "no-cond-assign": [2, "always"],
+    "no-console": 0,
+    "no-constant-condition": 2,
+    "no-control-regex": 2,
+    "no-debugger": 2,
+    "no-dupe-args": 2,
+    "no-dupe-keys": 2,
+    "no-duplicate-case": 2,
+    "no-empty-character-class": 2,
+    "no-empty": 2,
+    "no-ex-assign": 2,
+    "no-extra-boolean-cast": 2,
+    "no-extra-semi": 2,
+    "no-func-assign": 2,
+    "no-invalid-regexp": 2,
+    "no-irregular-whitespace": 2,
+    "no-negated-in-lhs": 2,
+    "no-obj-calls": 2,
+    "no-regex-spaces": 2,
+    "no-sparse-arrays": 2,
+    "no-unexpected-multiline": 2,
+    "no-unreachable": 2,
+    "no-underscore-dangle": 0,
+    "quotes": [2, "single"],
+    "semi": [2, "always"],
+    "space-before-blocks": [2, "always"],
+    "space-before-function-parentheses": [0, "never"],
+    "space-in-brackets": [0, "never"],
+    "space-in-parens": [2, "never"],
+    "space-unary-ops": [1, { "words": true, "nonwords": false }],
+    "strict": [2, "never"],
+    "use-isnan": 2,
+
+    // style
+    "curly": 2,
+    "dot-location": [2, "property"],
+    "dot-notation": 2,
+    "eqeqeq": 2,
+    "no-alert": 2,
+    "no-else-return": 2,
+    "no-eval": 2,
+    "no-extra-bind": 2,
+    "no-fallthrough": 2,
+    "no-floating-decimal": 2,
+    "no-implied-eval": 2,
+    "no-iterator": 2,
+    "no-labels": 2,
+    "no-lone-blocks": 2,
+    "no-loop-func": 0,
+    "no-multi-spaces": 2,
+    "no-multi-str": 2,
+    "no-native-reassign": 2,
+    "no-new-func": 2,
+    "no-new-wrappers": 2,
+    "no-new": 2,
+    "no-octal-escape": 2,
+    "no-octal": 2,
+    "no-proto": 2,
+    "no-redeclare": 2,
+    "no-return-assign": 2,
+    "no-self-compare": 2,
+    "no-sequences": 2,
+    "no-throw-literal": 1,
+    "no-unused-expressions": 2,
+    "no-useless-call": 2,
+    "no-useless-concat": 2,
+    "no-void": 2,
+    "no-warning-comments": [1, { "terms": ["todo", "fixme", "xxx"], "location": "start" }],
+    "no-with": 2,
+    "yoda": 2,
+    "no-delete-var": 2,
+    "no-shadow-restricted-names": 2,
+    "no-shadow": [0, { "hoist": "never" }],
+    "no-undef-init": 2,
+    "no-undef": 2,
+    "no-unused-vars": 2,
+    "callback-return": 2,
```

Comment:
Could you add the `eol-last` rule and update the `callback-return` rule not warn against the `next` keyword? Here's a patch that did those two things to `reddit-mobile` https://github.com/reddit/reddit-mobile/commit/a517659b804ba93134108377e2f1573765940b01#diff-1dc6ee56b778cd91e0327b52aaeaa1b9R101